DSL/DataMapper/lib/helpers.js

```diff
@@ -24,6 +24,13 @@ Handlebars.registerHelper('extractSlotKeys', function(obj) {
     return keys;
 });
 
+Handlebars.registerHelper('getObjectKeys', function(obj) {
+    console.log('OBJECT')
```

Comment:
Please remove `console.log`

Reply:
@ExiRain, did you make the change Ahmed wrote in the comment?

Reply:
Fixed in https://github.com/buerokratt/Training-Module/pull/355